### ISSUE TITLE
Node + icons do not appear in Explorer Navigation Pane

### ISSUE THREAD

**Issue by BlohoJo** [malvinas2/ClassicThemeForWindows10#7]
I'm using **Theme For Win10 November 2019 Update 1909**.

I ***really*** appreciate your theme replacing the stupid Windows 10 > node icons with the + ones, which work in most places, like the Registry Editor.

However, they don't appear in the Explorer Navigation pane:

![navpane10](https://user-images.githubusercontent.com/8242464/101453146-dc11cc00-38fc-11eb-90fc-e7421147644f.png)

The Navigation pane works, but the + node icons are invisible.

Any way to get them back? It's not a huge deal but it's really bugging me. :smile: 

I was on Windows 10 Pro x64 1909. I updated to 2004 and the theme still works. The + node icons on the Navigation Pane are still invisible however! :stuck_out_tongue: 

This is Windows 7 for comparison:

![navepane7](https://user-images.githubusercontent.com/8242464/101453636-a5888100-38fd-11eb-9261-8415897dfdd5.png)

**Comment by malvinas2** [comment]
I fear you've found another issue as I experience the opposite phenomenon: 
I can't change from plus-sign to triangle. 
Normally one can change between the two node styles using "Classic Explorer Settings" from OpenShell, see the attached screenshot (which shows issue nr. 7, check marks are invisible using the Classic Theme). But no matter what option I choose, nothing gets changed, the plus-sign is always shown here. It does work if you use another theme, I verified that, so it has to do something with Classic Theme. 
![node](https://user-images.githubusercontent.com/25912063/101546708-7deef280-39a9-11eb-9c55-143f1f710d09.jpg)

P.S.: Regedit and Explorer are two different things.

**Comment by BlohoJo** [comment]
Thanks for the reply!

I have Open-Shell 4.4.160. My check marks are visible as radio buttons using your theme. I have "Windows XP Simple" selected:

![YvzChYD](https://user-images.githubusercontent.com/8242464/101594576-ecd74600-39bf-11eb-94b1-cc6086719b4a.png)

But I FORGOT to mention... I am using [Simple Classic Theme](https://github.com/AEAEAEAE4343/SimpleClassicTheme), along with yours for scrollbars and 3D effects. Using this combination I can get Windows 10 very close to a "XP Classic" look! (Except for the Navigation Pane nodes!) :stuck_out_tongue_closed_eyes:

![sctt_about](https://user-images.githubusercontent.com/8242464/101595402-4c822100-39c1-11eb-8b69-0809578e8445.png)

**Comment by malvinas2** [comment]
Using SimpleClassicTheme I can confirm the issue: No signs appear in the navigation pane, but check marks in dialog boxes of OpenShell. But generally, mixing up a Visual Style and a 'real' Classic Theme is not a good idea and I can't support this combination.

**Comment by BlohoJo** [comment]
I just recently discovered (after I posted here) that it's actually only when SCT is started do the nodes disappear. So your theme is fine! I apologize for that. (I swore I tried it with SCT disabled earlier... maybe I forgot to kill and restart explorer.exe properly.)

Wish is was something simple that could be fixed without taking too much time or effort.

I'll live with it... I haven't found any other msstyles theme that works as good as yours does in getting the Windows Classic look and feel right. Combining SCT with msstyles may not be the best idea but it's been pretty stable so far!
